modules/shibboleth/templates/shibboleth2.xml.erb

```diff
@@ -65,6 +65,14 @@
                     <md:GivenName>Unity SSO Support</md:GivenName>
                     <md:EmailAddress>unity-sso@researchresearch.com</md:EmailAddress>
                 </md:ContactPerson>
+
+                <mdui:UIInfo>
+                    <mdui:DisplayName xml:lang="en">*Unity collaboration environment</mdui:DisplayName>
+                    <mdui:Description xml:lang="en">*Research's OAE implementation for educational institutions</mdui:Descriptione>
```

Comment:
This can either be `Where Universities work together` or `*Unity is the new way to work with anyone in universities and research. It’s a simple place to form groups, share files and work on them together.`